authors/apps/authentication/serializers.py

```diff
@@ -13,7 +15,10 @@ class RegistrationSerializer(serializers.ModelSerializer):
     password = serializers.CharField(
         max_length=128,
         min_length=8,
-        write_only=True
+        write_only=True,
+        error_messages={
```

Comment:
@wasibani-roy , since the minimum password length is 8 characters,we could have a message like 
> password should be at least 8 characters long

Reply:
Okay worked on it